tests/unit/test_xasprintf_x.c

```diff
@@ -0,0 +1,81 @@
+/*
+ * SPDX-FileCopyrightText: 2023, Alejandro Colomar <alx@kernel.org>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+
+#include <setjmp.h>
+#include <stddef.h>
+#include <stdlib.h>
+
+#include <stdarg.h>  // Required by <cmocka.h>
+#include <stddef.h>  // Required by <cmocka.h>
+#include <setjmp.h>  // Required by <cmocka.h>
+#include <stdint.h>  // Required by <cmocka.h>
+#include <cmocka.h>
+
+#include "sprintf.h"
+
+
+#define assert_unreachable()  assert_true(0)
+
+
+static jmp_buf  jmpb;
+
+
+/**********************
+ * WRAPPERS
+ **********************/
+int
+__wrap_vasprintf(char **restrict p, const char *restrict fmt, va_list ap)
+{
+	return -1;
+}
+
+
+void
+__wrap_exit(int status)
+{
+	longjmp(jmpb, 42);
+}
+
+
+/**********************
+ * TEST
+ **********************/
+static void
+test_xasprintf_x(void **state)
```

Comment:
Rename the test case to something more meaningful like `test_xasprintf_unhappy` or `test_xasprintf_error` or `test_xasprintf_exit`.

Reply:
(The rationale for just `x` was because I was testing for the x in _x_asprintf().  :)

But yeah, could do that.